brax/envs/wrappers.py

```diff
@@ -55,3 +56,119 @@ def reset(self):
   def step(self, action):
     self._state, obs, reward, done = self._step(self._state, action)
     return obs, reward, done, {}
+
+  def seed(self, seed: int = 0):
```

Comment:
Thanks for adding this method - please call it from the initializer like you do in the other class.